homeassistant/components/hassio/handler.py

```diff
@@ -100,52 +91,6 @@ async def async_create_backup(
     return await hassio.send_command(command, payload=payload, timeout=None)
 
 
-@bind_hass
-@api_data
-async def async_update_os(hass: HomeAssistant, version: str | None = None) -> dict:
-    """Update Home Assistant Operating System.
-
-    The caller of the function should handle HassioAPIError.
-    """
-    hassio: HassIO = hass.data[DOMAIN]
-    command = "/os/update"
-    return await hassio.send_command(
-        command,
-        payload={"version": version},
-        timeout=None,
```

Comment:
For reviewers - yes this change remove these overrides of the default timeout of 10s. I discussed it with Martin and all these timeout overrides for specific methods will be incorporated in the library. It will set appropriate timeouts for each API call for clients instead of asking them for a default timeout. This change will be made in https://github.com/home-assistant-libs/python-supervisor-client/pull/23 and incorporated into the next release.